src/diffusers/schedulers/scheduling_heun.py

```diff
@@ -131,7 +133,7 @@ def set_timesteps(
         self.init_noise_sigma = self.sigmas.max()
 
         timesteps = torch.from_numpy(timesteps)
-        timesteps = torch.cat([timesteps[:1], timesteps[1:].repeat_interleave(2), timesteps[-1:]])
```

Comment:
This was incorrect previously, but didn't change the output of the image. Can be removed so since it's an unnecessary extra step